### PR TITLE
Bump versions for 3.20.0 final

### PR DESCRIPTION
**Pull request by AlvaroBrey**
- [x] https://github.com/nextcloud/android/pull/10144 merged
- [x] https://github.com/nextcloud/android/pull/10145 merged
- [x] Tests written, or not not needed

### CODE REVIEW

**Review by AndyScherzinger — APPROVED** [review]
LGTM

File: fastlane/metadata/android/en-US/changelogs/30200090.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## 3.20.0 (May 2nd, 2022)
```
Comment (tobiasKaminsky) [inline fastlane/metadata/android/en-US/changelogs/30200090.txt:1]: 3.05. :)

Comment (AlvaroBrey) [reply fastlane/metadata/android/en-US/changelogs/30200090.txt]: :facepalm:  fixed

**Review by tobiasKaminsky — APPROVED** [review]

**Pull request merged**